gc.rb

```diff
@@ -164,7 +191,8 @@ def self.latest_gc_info hash_or_key = nil
   end
 
   def self.compact
-    Primitive.rb_gc_compact
+    Primitive.gc_start_internal true, true, true, true
+    Primitive.gc_compact_stats
```

Comment:
Where does `Primative` come from? It's a pretty nice interface.

Reply:
This is new in master. I *think* ko1 introduced it.  I'm very excited about `Primitive`

Reply:
So it can invoke C  functions?

Reply:
FWIW it's from https://bugs.ruby-lang.org/issues/16254 + my suggestion to rename to `Primitive.name` like in TruffleRuby (and similar in Rubinius before that).